docs/scripting/gametest-form.md

```diff
@@ -242,9 +246,9 @@ form.toggle("Hide Effect Particle", true)
 ![image](/assets/images/gametest/gametest-form/modal-form.png)
 
 ## Show and Respond
-After we create the form, we need to show the form to the player and save the response to run another task. We need some event to show our form. The most used event is using `itemUse` event, that read when player use (right-click) an item.
+After we create the form, we will need to show the form to the player and save the response to run other tasks. We will need some event to show our form. The most used event is using `itemUse` event, that reads when player uses (right-clicks) with an item.
```

Comment:
"reads when player uses with an item" does not make sense.
Consider: 
- that reads when a player uses (right-clicks) an item 
- that reads when players use (right-clicks) an item